source/Ship.cpp

```diff
@@ -2226,7 +2225,6 @@ int Ship::TakeDamage(const Projectile &projectile, bool isBlast)
 	}
 	
 	// Recalculate the disabled ship check.
-	isDisabled = true;
```

Comment:
`isDisabled` is required to be set `true` before the evaluation of `IsDisabled()` since a full check for the disabled status is desired.